libpod/oci.go

```diff
@@ -433,8 +437,8 @@ func (r *OCIRuntime) execContainer(c *Container, cmd, capAdd, env []string, tty
 		args = append(args, "--no-new-privs")
 	}
 
-	for _, cap := range capAdd {
-		args = append(args, "--cap", cap)
+	for _, capabilityAdd := range capAdd {
+		args = append(args, "--capabilityAdd", capabilityAdd)
```

Comment:
Woah - we're changing a flag name here - that doesn't seem right?

Reply:
Confirmed in `runc` - `--cap` is a flag, `--capabilityAdd` is not

Reply:
nice catch!